bundles/org.openhab.binding.konnected/README.md

```diff
@@ -28,24 +28,33 @@ The blink setting will disable the transmission LED on the Konnected Alarm Panel
 
 ## Channels
 
-The auto discovered thing adds two default channels.
+Depending on whether the thing type is the wifi version or the pro version the automativally added channels will differ.  
+For the wifi verision the auto discovered thing adds two default channels.
 
 | Channel | Channel Id | Channel Type | Description                                              |
 |---------|------------|--------------|----------------------------------------------------------|
-| 1       | Zone_6     | Switch       | A Switch channel for zone 6                              |
-| 2       | Out        | Actuator     | The Channel for the Out Pin on the Konnected Alarm Panel |
+| 1       | Zone_6     | Switch-wifi       | A Switch channel for zone 6                              |
+| 2       | Out        | Actuator-wifi     | The Channel for the Out Pin on the Konnected Alarm Panel |
```

Comment:
What do you mean by "Channel Type"? I think it could be easily mixed up with the Item type. Channel names should be all lower case.

Reply:
When the things are added you create channels depending on what sensors you have connected to the zones on the board.  The binding provides the types of channels which can be added but the user must actually create the channels on the thing itself.